serverless/serverless
# Add metrics plugin

## What did you implement:

Closes #1650

Add a metrics plugin so that specific metrics for a function can be retrieved.

## How did you implement it:

Add core `metrics` and AWS implementation of the `metrics` plugin.

## How can we verify it:

Deploy a service, invoke a function and then run `serverless metrics --function hello` to see the metrics of this function.

## Todos:


***Is this ready for review?:*** YES
